src/main/resources/application.properties

```diff
@@ -1 +1,17 @@
+# Database connection settings
+spring.datasource.url=jdbc:mysql://localhost:3306/liftoff
```

Comment:
whats the difference between this file and the one here build/resources/main/application.properties???